### PR TITLE
7746 - Search bar redesign

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

This PR updates the documentation and images about filters and search redesign.

https://github.com/medic/cht-core/issues/7746
[CHT Core PR](https://github.com/medic/cht-core/pull/7758)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
Hi @michaelkohn can you please review this doc? 
The images are quite of a work to make 😅 I hope they are okay

Video:

https://user-images.githubusercontent.com/66472237/189071530-198e2feb-34b3-4216-8265-9daf7fd468b4.mov

File: content/en/apps/features/reports/_index.md [hunk 1]
```diff
@@ -37,15 +37,15 @@ The toolbar at the top of the page includes filters and search to help users nar
...
+{{< figure src="reports-search.png" link="reports-search.png" class="left col-3 col-lg-3" >}}
 
-{{< figure src="reports-search.png" link="reports-search.png" class="left col-9 col-lg-9" >}}
+{{< figure src="reports-search-reset.png" link="reports-search-reset.png" class="left col-9 col-lg-9" >}}
 
 <br clear="all">
 
-{{% alert title="Note" %}} A new user experience for Filter and Search was introduced in v3.17. The previous version can be re-enabled for users by adding the [permission]({{< ref "apps/reference/app-settings/user-permissions" >}}) `can_view_old_reports_filter` to the user's role; however, the old version should be considered deprecated and will be completely removed in a future release. {{% /alert %}}
+{{% alert title="Note" %}} A new user experience for Filter and Search was introduced in v3.17. The previous version can be re-enabled for users by adding the [permission]({{< ref "apps/reference/app-settings/user-permissions" >}}) `can_view_old_filter_and_search` to the user's role; however, the old version should be considered deprecated and will be completely removed in a future release. {{% /alert %}}
```
Comment (michaelkohn) [inline content/en/apps/features/reports/_index.md:48]: How about this?

....however, the old version should be considered deprecated and will be completely removed in a future release. See [Feature Flags](https://docs.communityhealthtoolkit.org/apps/guides/updates/feature-flags/) documentation for more info.

File: content/en/apps/reference/app-settings/user-permissions.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ Permissions can be assigned to user roles either directly in `base_settings.json
 | `can_view_unallocated_data_records` | Allows viewing reports that have no associated contact |
 | `can_view_users` | Allows viewing all user accounts |
 | `can_write_wealth_quintiles` | Allows updating contacts with wealth quintile information |
-| `can_view_old_reports_filter` | Allows users to see the old filter in Reports Tab which is considered deprecated and will be completely removed in a future release. Admin user will always see the new redesigned filter. |
+| `can_view_old_filter_and_search` | Allows users to see the old filter and search in Reports Tab and Contact Tab which is considered deprecated and will be completely removed in a future release. Admin user will always see the new redesigned filter. |
```
Comment (michaelkohn) [inline content/en/apps/reference/app-settings/user-permissions.md:68]: Maybe add: "See [Feature Flags](https://docs.communityhealthtoolkit.org/apps/guides/updates/feature-flags/) for more info."

File: content/en/apps/features/contacts/_index.md [hunk 3]
```diff
@@ -46,11 +46,11 @@ With the [_UHC Mode_]({{< relref "apps/features/uhc-mode" >}}) configured, the m
...
+Click in the search icon at the top of the screen to search for a “person” or “place”. The freetext search works on all fields included in the “person” or “place” document such as patient name or patient ID. The exact fields depends on which information you’ve configured your app to collect.
 
-After entering a search term, the list filters to show matching items. Searching will only return items that are lower than you in the hierarchy and that you have permission to view. 
+After typing a search term, press the "Enter" key on your keyboard, then the list filters to show matching items. Searching will only return items that are lower than you in the hierarchy and that you have permission to view. 
 
-To clear the search and return the default view, click on the refresh icon located to the right of the search box.
+To clear the search and return the default view, click on the arrow icon located to the left of the search box.
```
Comment (michaelkohn) [inline content/en/apps/features/contacts/_index.md:53]: **`Instead of:`** _return the default view_
**`Better:`** _return **to** the default view_

**Review by michaelkohn — COMMENTED** [review]
Two little grammar things that I'd recommend, and a couple other suggestions that I'm totally fine with not doing if you prefer not to.  Otherwise looks great!

**Review by michaelkohn — APPROVED** [review]
LGTM!

**Pull request merged**